src/Raven.Server/Documents/PeriodicBackup/Restore/RestoreBackupTaskBase.cs

```diff
@@ -343,7 +344,7 @@ public async Task<IOperationResult> Execute(Action<IOperationProgress> onProgres
 
                     result.AddInfo($"Loading the database after restore");
                     var databasesLandlord = this._serverStore.DatabasesLandlord;
-                    _ = databasesLandlord.TryGetOrCreateResourceStore(databaseName);
+                    _ = databasesLandlord.TryGetOrCreateResourceStore(databaseName).IgnoreUnobservedExceptions();
```

Comment:
We need to include the db load in the restore output as well

Reply:
Do we want to wait for the database to load as part of the restore process?

Reply:
In the issue, it says that the load is part of the restore, no? So it should wait I assume.
Otherwise, the DB isn't ready